### PR TITLE
FISH-6103 SimplePolicyProvider cannot be used for JACC Per Application

### PR DESCRIPTION
**Pull request by jGauravGupta**
This PR wraps exousia Policy Provider and provides JACC per application feature by implementing JaccConfigurationFactory

### CODE REVIEW

**Comment by jGauravGupta** [comment]
jenkins test please

**Comment by jGauravGupta** [comment]
jenkins test please

File: appserver/security/jacc.provider.inmemory/src/main/java/fish/payara/security/jacc/provider/PolicyConfigurationFactoryImpl.java [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+//        SimplePolicyConfiguration policyConfigurationImpl = null;
+//        
+//        File contextDirectory = new File(getContextDirectoryName(contextId));
+//        if (contextDirectory.exists()) {
+//            policyConfigurationImpl = new PolicyConfigurationImpl(contextDirectory, open, remove, this);
+//            contextToConfigurationMap.put(contextId, policyConfigurationImpl);
+//        }
+//        
+//        return policyConfigurationImpl;
+//    }
```
Comment (Pandrex247) [inline appserver/security/jacc.provider.inmemory/src/main/java/fish/payara/security/jacc/provider/PolicyConfigurationFactoryImpl.java:250]: Dead code?

**Review by Pandrex247 — APPROVED** [review]
Some dead code hanging around but otherwise looks good 👍

**Comment by jGauravGupta** [comment]
jenkins test please

**Comment by jGauravGupta** [comment]
jenkins test please

**Pull request merged**